srv/salt/ceph/updates/default-zypper-kernel-default.sls

```diff
@@ -0,0 +1,22 @@
+
+
+zypper update:
+  cmd.run:
+    - name: "zypper --non-interactive  update --replacefiles --auto-agree-with-licenses"
+    - shell: /bin/bash
+    - unless: "zypper lu | grep -sq 'No updates found'"
+
+kernel install:
+  cmd.run:
+    - name: "zypper --non-interactive --no-gpg-checks install --force-resolution kernel-default"
+    - shell: /bin/bash
+    - unless: "rpm -q kernel-default"
+    - fire_event: True
+
+kernel update:
```

Comment:
@rjfd With respect to the issue in  #60, I can but it's not at the top of the list for me.  I am blocked because I do not have a Salt way of changing the kernel.

As for the 'rpm -q' is the least expensive way to make this idempotent.  The zypper commands quite literally take longer even when doing nothing and output results that tells Salt that something has changed.  The ultimate goal is that all of this will "go away" and be handled by the Salt zypper module.  In the meantime, I need a method to provision my vagrant box with the desired kernel and since I am sharing this with others, they will also need this so I do not think I should keep the change to myself.